manage/users/index.rst

```diff
@@ -161,29 +161,30 @@ The ones that aren’t are described below.
 
          .. _user-details-group-access:
 
-🔓 Permissions
-   Under this heading, you can manage two separate (but related) user details:
-
-   1. :doc:`/manage/roles/index` dictate what users can do in the system.
-      If you need to grant someone privileges to edit the knowledge base
-      or access part of the admin panel, roles are the answer.
+Roles
+   The :doc:`/manage/roles/index` define what users can do in the system.
+   If you need to grant someone privileges to edit the knowledge base
+   or access part of the admin panel, roles are the answer.
 
-   2. :doc:`/manage/groups/access-levels` dictate which tickets
-      an agent can work with.
-      If someone’s not receiving notifications for incoming tickets
-      or can’t be assigned a ticket,
-      group access levels are likely to blame.
+Group Permissions
+   The :doc:`/manage/groups/access-levels` define which tickets
+   an agent can work with.
+   If an agent is not receiving notifications for incoming tickets
+   or can't be assigned to a ticket, you should have a look on the group
+   permissions.
+   Please note that the visibility of the group permission table depends on the
+   role selection - it has to be "Agent" that the table shows up.
 
    .. figure:: /images/manage/users/user-details-permissions.png
       :alt: Permissions in the edit user dialog
       :align: center
       :width: 70%
 
-      **Top:** A user’s roles decide what kinds of actions they can perform
-      and which :doc:`groups </manage/groups/index>` they belong to.
-      **Bottom:** Group assignments can alternately be set on a per-user basis.
+   **Top:** A user's roles decide what kinds of actions they can perform
```

Comment:
`A user's roles what kinds of actions` => `User's roles decide what kind of actions`